content/en/observability_pipelines/installation.md

```diff
@@ -60,7 +61,13 @@ $ DD_API_KEY=<DD_API_KEY> DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.
     $ sudo apt-get install datadog-observability-pipelines-worker datadog-signing-keys
     ```
 
-4. Start the Worker:
+4. Create a SystemD service file using the [vector.service][7] example as a reference:
```

Comment:
```suggestion
4. Create a SystemD service file using this command:
```

Reply:
I removed the link to the Vector docs because we're trying to separate out Vector and the Observability Pipelines Worker. Is there a reference we can use for the OPW specifically? If so, we can add that instead.